{{cookiecutter.project_slug}}/.osparc/runtime.yml

```diff
@@ -4,6 +4,6 @@ settings:
     type: Resources
     value:
       Limits:
-        NanoCPUs: 1 # 100% of CPU cycles on 1 CPU 
+        NanoCPUs: 1000000000 # 100% of CPU cycles on 1 CPU 
```

Comment:
yep ... nano :-)

Reply:
Eheh, indeed. Docker didn't like 1e-9 CPU as resource. I was confused by a conversation I had with SAN, we were thinking that the number represented "CPU cycles": https://docs.docker.com/config/containers/resource_constraints/#configure-the-default-cfs-scheduler